components/input/ColorPicker.css

```diff
@@ -0,0 +1,11 @@
+.popover {
+    position: absolute;
+    z-index: 2;
+}
+.cover {
+    position: fixed;
+    top: 0px;
```

Comment:
If the value is `0`, there is no need to have a unit ;)

Reply:
I suspected that. Let me change it!